pkg/lwip/contrib/sock/tcp/lwip_sock_tcp.c

```diff
@@ -380,13 +380,16 @@ ssize_t sock_tcp_write(sock_tcp_t *sock, const void *data, size_t len)
 }
 
 #ifdef SOCK_HAS_ASYNC
-void sock_tcp_set_cb(sock_tcp_t *sock, sock_tcp_cb_t cb)
+void sock_tcp_set_cb(sock_tcp_t *sock, sock_tcp_cb_t cb, void *arg)
```

Comment:
Why do you provide `arg` to `sock_tcp_set_cb()` when it's not used?

Reply:
Oops, this is an error from when I progressively introduced this in my commit history.

Reply:
Interesting, that the test still seems to pass though oO

Reply:
It didn't (probably did not clean properly on my last run). Fixed in latest PR